samples/bluetooth/fast_pair/locator_tag/README.rst

```diff
@@ -161,32 +244,9 @@ The user interface of the sample depends on the hardware platform you are using.
          * Blinks at a 1 second interval if the recovery mode is active on the device and the identification mode is inactive on the device.
 
       Button 1:
-         Toggles between different modes of the Fast Pair advertising set:
-
-         * Without an Account Key stored on the device:
-
-           * Fast Pair advertising disabled.
-           * Fast Pair discoverable advertising (the default after the system bootup).
-
-         * With an Account Key stored on the device:
-
-           * Fast Pair advertising disabled.
-           * Fast Pair not discoverable advertising (the default after the system bootup).
-
-         .. note::
-
-            The Bluetooth advertising is active only until the Fast Pair Provider connects to a Bluetooth Central.
-            After the connection, you can still switch the advertising modes, but the switch will come into effect only after disconnection.
-
-            The sample automatically switches to the following Fast Pair advertising modes under certain conditions:
-
-            * Fast Pair not discoverable advertising - On the Account Key write operation.
-            * Fast Pair advertising disabled:
-
-               * Right before the factory reset operation.
-                 Press **Button 1** to set the device to the Fast Pair discoverable advertising mode after the unprovisioning operation.
-
-               * After the beacon clock synchronization.
+         Requests turning on the Fast Pair advertising or removes such a request.
```

Comment:
```suggestion
         Either requests to turn on Fast Pair advertising or removes the request.
```
no strong feelings about this one - please use whichever works better

Reply:
Not that strong feeling either but I think I prefer the original sentence, so let's leave it.

Reply:
Got it - then keeping the original flow of your sentence just wanted to add one suggestion for better readability:

```suggestion
         Sends a request to turn on Fast Pair advertising or removes such a request.
```

Reply:
Sure, seems fine, applied for each target